terraform-unity/modules/terraform-unity-sps-eks/main.tf

```diff
@@ -9,8 +9,10 @@ terraform {
 }
 
 module "unity-eks" {
-  source          = "git@github.com:unity-sds/unity-cs-infra.git//terraform-unity-eks_module?ref=unity-sps-2.2.0-hotfix"
+  source          = "git::https://github.com/unity-sds/unity-cs-infra.git//terraform-unity-eks_module?ref=59-sps-eks-marketplace-adjustments"
```

Comment:
At present, this is pinned to a temporary tag (`59-sps-eks-marketplace-adjustments`), but I'm going to work with Galen on getting those changes into main, hopefully in the next day or two.

Reply:
right, these changes have been merged (and `59-sps-eks-marketplace-adjustments` cleaned up/deleted). We'll need to ask CS for a new tag/make a new tag for ourselves to track on before I can fix/merge this PR for good, I think.

Reply:
updated to new tag!